exercises/G-conditionals-2/exercise-3.js

```diff
@@ -8,7 +8,20 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+function calculateGrade(mark) {
+  if (mark >= 80) {
+  return "A"
+  }
+  else if (mark < 80 && mark > 60) {
+  return "B"
+  } 
+  else if (mark <= 60 && mark > 50) {
+  return "C"
+  } 
+  else  {
+  return "F"
+  } 
+}
 
```

Comment:
Hi Bisi , everything looks good and well-strutcured to me. I have some comments regarding G-conditionals/exercise-3 .
Lines 15 and 18 have some redundant conditions can you spot and remove them?
Here is some clue: test for mark= 61 and mark=51
Well done!